### PR TITLE
BBE :  WebsiteContent : Extra images for content pages.

### PR DESCRIPTION
**Pull request by ddc22**
#### Proposed Changes

* Context pdh1Xd-1yv-p2
* Allows 4 images for all pages
* Image width sizes were reduced
image

* Allows 8 images for portfolio
image

#### Testing Instructions

* Go through the DIFM flow and complete it by purchasing all pages (`/start/do-it-for-me` )
* On the website content page make sure above changes are visible

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Fixes Automattic/martech#1232
Fixes Automattic/martech#1233

### CODE REVIEW

**Review by aneeshd16 — COMMENTED** [review]
Works well! I have a suggestion regarding the design:

Adding the extra image boxes increases the width of the form and would be a change in the design.

**Before**
image

**After**
image

What if we decrease the size of the image upload boxes proportionally so that the width of the form remains the same?
**Suggested**
image

**Comment by ddc22** [comment]
@aneeshd16 
This is something that I raised ( pdh1Xd-1yv-p2 ) I feel like expanding the content side is better UX for the user that is why I left it like that. WDYT?

File: client/signup/accordion-form/accordion-form-section.tsx [hunk 1]
```diff
@@ -22,6 +22,9 @@ interface SectionHeaderProps {
 const Section = styled.div`
 	border-bottom: 1px solid var( --studio-gray-5 );
 	padding: 0 20px;
+	@media ( min-width: 600px ) {
+		width: 775px;
+	}
```
Comment (aneeshd16) [inline client/signup/accordion-form/accordion-form-section.tsx:27]: Can you please help me understand why this change was required? 🤔

File: client/signup/accordion-form/form-components.tsx [hunk 2]
```diff
@@ -246,6 +246,6 @@ export const HorizontalGrid = styled.div`
 	margin-bottom: 20px;
 	flex-wrap: wrap;
 	@media ( min-width: 1100px ) {
-		flex-wrap: nowrap;
+		flex-wrap: wrap;
```
Comment (aneeshd16) [inline client/signup/accordion-form/form-components.tsx:249]: The default value is already `wrap`, so this change has no effect.

Comment (aneeshd16) [inline client/signup/accordion-form/form-components.tsx]: Any reason for this change? The `flex-wrap: nowrap;` could be there since we used this component before in another place. If `flex-wrap: nowrap;` is not required anymore, we can simply remove this media query since the default value for `flex-wrap` is `wrap` (L247)

**Review by aneeshd16 — APPROVED** [review]
Works well! I left some minor questions below, but overall LGTM!

File: client/signup/accordion-form/form-components.tsx (hunk 2 above)
Comment (ddc22) [reply client/signup/accordion-form/form-components.tsx]: Ah yes nice catch let me check this

File: client/signup/accordion-form/accordion-form-section.tsx (hunk 1 above)
Comment (ddc22) [reply client/signup/accordion-form/accordion-form-section.tsx:27]: This is tied to the flex wrap. By providing a concrete width I am forcing the image upload placeholders to wrap to the next line beyond a certain width. :)

File: client/signup/accordion-form/form-components.tsx (hunk 2 above)
Comment (ddc22) [reply client/signup/accordion-form/form-components.tsx]: I have checked the other places as well. They are used in the same way. So this part can be removed 👌

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7624766

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7624766).
